### DISCUSSION TITLE
Are there any mistake in this div tag?

### DISCUSSION THREAD

**Discussion by capraynor** [photoprism/photoprism#3992] [Q&A] [answered]
Reading PP code in these days, trying to create timeline feature. 

Are there any mistake in this tag? The div tag should not be self-closed. 

If you think that should not be like this please tell me, I'll modify it. 

https://github.com/photoprism/photoprism/blob/cc356abe03dd4bdfa9ac60472e0b6b04ec96e96c/frontend/src/component/photo/list.vue#L56

**Comment by capraynor** [comment]
<img width="685" alt="image" src="https://github.com/photoprism/photoprism/assets/3467515/ecc81fc0-d0b5-45ac-8cf6-45fecd97caf1">

Sorry may be this is a special usage in Vue (v-if and v-else), is that true?

**Comment by lastzero** [comment, marked as answer]
It's obviously not correct based on the HTML standard, but might still work since all templates and JS are transpiled before they are displayed in a browser? Feel free to change this.

**Reply by lastzero** [reply]
Should be fixed with this commit: https://github.com/photoprism/photoprism/commit/b259d8fcd35a0bc2e6d41aaee22c2cdff39855b7

**Reply by capraynor** [reply]
>Feel free to change this.

OK got it.
 
And, 

Thank you for putting this into discussion, I'll start this kind of topic in github discussion next time.
